### PR TITLE
feat(ErrorLogger): individual error state for expansion - I17

### PR DESCRIPTION
**Pull request by irmerk**
# Issue #17
Modularize individual error component for stateful expansion

### Changes
- Increased the click area for error expansion
- Modularize and migrate component logic for individual state handling

### Flags
- Click area could still use some work, which will be a future issue for @Michael-Grover

### Related Issues
- Pull Request #101
- Pull Request #98

### CODE REVIEW

File: src/ErrorLogger/error.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/* React */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+/* Styling */
```
Comment (DianaLease) [inline src/ErrorLogger/error.js]: Styling and Actions comments are switched

File: src/ErrorLogger/index.js [hunk 2]
```diff
@@ -8,21 +8,19 @@ import * as A from './actions';
 /* Actions */
```
Comment (DianaLease) [inline src/ErrorLogger/index.js]: should be Styles

**Comment by irmerk** [comment]
Still looking for a better method for rebasing, the one I have now is not working.

**Comment by irmerk** [comment]
> I think you may need to do something like this to get git to recognize the file name casing change: https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git/40307511

That would have been easier, I did a file name swap.

**Pull request merged**